packages/tslint/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Tslint webpack block.
+ *
+ * @see https://github.com/wbuchwalter/tslint-loader
+ */
+
+module.exports = tslint
+
+/**
+ * @param {object}   [options]                  See https://github.com/wbuchwalter/tslint-loader#usage
+ * @return {Function}
+ */
+function tslint (options) {
+  options = options || {}
+
+  const loader = context => ({
+    test: context.fileType('application/x-typescript'),
+    loaders: [ 'tslint-loader' ]
+  })
+
+  const module = context => (context.webpackVersion.major === 1 ? {
```

Comment:
`standard` usually requires you to put a single parameter of an arrow function into parenthesis. Maybe the linting of this code does not work? 🤔